CRM/Bsd/Page/BSD.php

```diff
@@ -36,6 +36,14 @@ class CRM_Bsd_Page_BSD extends CRM_Core_Page {
 
   public $new_contact = false;
 
+  private $apiAddressGet = 'api.Address.get';
```

Comment:
not a big fan of that: it doesn't make the code sorter, just adding another "name indirection" that makes it more confusing to read.